domains/transfer/Transfer.API/API.IntegrationTests/Controllers/TransferAgreementsControllerTest.cs

```diff
@@ -159,6 +159,35 @@ public async Task Create_ShouldFail_WhenReceiverTinInvalid(string tin, string ex
         validationProblemContent.Should().Contain("ReceiverTin");
     }
 
+    [Theory]
+    [InlineData("This is not a valid Base64 string", false, "Base64-encoded Wallet Deposit Endpoint is not valid")]
+    [InlineData("U28gbG9uZyBhbmQgdGhhbmtzIGZvciBhbGwgdGhlIGZpc2gu", true, null)]
+    [InlineData("=", false, "Base64-encoded Wallet Deposit Endpoint is not valid")]
+    [InlineData("12345", false, "Base64-encoded Wallet Deposit Endpoint is not valid")]
+    [InlineData("YWJjZA==", true, null)]
+    public async Task Create_ShouldValidate_WalletDepositEndpoint_ToBeBase64Encoded(string base64String, bool shouldSucceed, string expectedMessage)
```

Comment:
I think you should split this test in two cases on with success and one with error then you can avoid the boolean

https://medium.com/@amlcurran/clean-code-the-curse-of-a-boolean-parameter-c237a830b7a3

Reply:
Refactored in https://github.com/Energinet-DataHub/energy-origin/pull/443/commits/8329052c0b43486a48e587d8e78678aed0c3d0f5